README.md

```diff
@@ -44,15 +44,15 @@ To build the jar files, you must use minimum version of Java 11 with Maven. You
 * Maven:
 	1. If you have not already done so, add the environment variable `mssql_jdbc_test_connection_properties` in your system with the connection properties for your SQL Server or SQL DB instance.
 	2. Run one of the commands below to build a JRE 11 and newer versions compatible jar or JRE 8 compatible jar in the `\target` directory. 
-        * Run `mvn install -Pjre20`. This creates JRE 20 compatible jar in `\target` directory which is JDBC 4.3 compliant (Build with JDK 20).
+        * Run `mvn install -Pjre21`. This creates JRE 20 compatible jar in `\target` directory which is JDBC 4.3 compliant (Build with JDK 20).
```

Comment:
Why not also update the text "creates JRE 21 compatible jar", "Build with JDK 21". Maybe even "Built"?

Reply:
Thanks. It was an accident. I did a text replace to update the file using the search term 'jre20'.